### PR TITLE
Support ref

### PR DESCRIPTION
**Pull request by monopieces**
A workaround solution to support  `ref`.

```
<Component innerRef={this.ref} />
```

### CODE REVIEW

File: __test__/__snapshots__/test.js.snap [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`<I18nProvider> children get i18n from I18nProvider 1`] = `
-<TestElement
+<UNDEFINED
```
Comment (SBoudrias) [inline __test__/__snapshots__/test.js.snap]: I think this `UNDEFINED` is an unintended side-effect. Can we fix this on our side?

Comment (monopieces) [reply __test__/__snapshots__/test.js.snap]: Fixed, thanks!

File: __test__/test.js [hunk 2]
```diff
@@ -68,9 +68,7 @@ describe('i18n hook', () => {
 describe('<I18nProvider>', () => {
     it('children get i18n from I18nProvider', () => {
         const LocalizedTest = translate(TestElement);
-        const WrappedTest = LocalizedTest.WrappedComponent;
-
-        expect(<WrappedTest testProp="required" i18n={mockI18n} />).toMatchSnapshot();
+        expect(<LocalizedTest testProp="required" i18n={mockI18n} />).toMatchSnapshot();
```
Comment (SBoudrias) [inline __test__/test.js]: Why are we changing this test? It doesn't test the same thing as before.

Comment (monopieces) [reply __test__/test.js]: Sorry, it is debugging code, I'll rollback this part.

**Pull request merged**